docs/testing.md

```diff
@@ -40,6 +40,10 @@ To run just a subset of them, specify a particular file or directory:
     py.test tests/unit
     py.test tests/unit/api.py
 
+To run just a pep8 test:
+
+    py.test --pep8 -m pep8 *
```

Comment:
@henn I went with `py.test --pep8 -m pep8 *.py haas/ tests/` because I like that we're explicitly calling out what we want to scan.  I think its probably the best because it reinforces is taking the folders as input and will be easy to understand for any new folks